### DISCUSSION TITLE
[PROPOSAL] SFLGpt collectible

### DISCUSSION THREAD

**Discussion by brian-man** [sunflower-land/sunflower-land#2292] [General]
Mini-proposal:
> SFLGpt
> Mintable supply: 11011
> Ingredients: 400 SF, 200 Potato, ... (the supply amount of every crop from SF to Kale)
> Boost: When clicked, disables "Reward" proof-of-human test for 1 hour.
> Cooldown: 1 day

**Comment by BlazinMazon** [comment]
I feel the boost should be permanent for VIP players or holders of certain SFT's. Like Davy Jones or the Treasure map one constant boost of no captchas certifying players as verified.
